.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,16 @@
+{
+	"name": "Foundry + Node",
+	"image": "mcr.microsoft.com/devcontainers/base:0",
+
```

Comment:
```suggestion
```

IDE autoformatting deletes this whitespace, so my weak preference would be to remove it here.